### PR TITLE
Fix work with Oracle and added new 'schema' attribute to Model's Meta

### PR DESCRIPTION
**Pull request by thakryptex**
## Description
- Updated the connection string for Oracle Client (DBQ key).
- Added new `schema` attribute to Model's Meta.
- Fixed ORA-01435 error while using Oracle database (#1155).
- Updated docs.

## Motivation and Context
- I had a problem, that I can't connect to existing Oracle database. It was throwing an error that database info is not provided or something like that. So I've checked how it was written and found that existing connection string for Oracle connection wasn't full. That lead me to the first fix: I updated the connection string for Oracle Client.
- Then I had a problem that I need to specify the schema that I'm working on (as for MsSQL and Oracle, because I have several DB). And I found in code that it was almost ready to set schema and properly paste it into queries. So few changes and that worked.
- Then I had a error from Oracle that user doesn't exists. I created an issue #1155. Then I finally found the reason of this message. And fixed it. It was putting database name into query, while schema name was needed. So I've passed user attribute instead of database attribute, because in Oracle database schema name is equal to a username.

## How Has This Been Tested?
I've run all of the existing tests and they all passed.
And I've tested this code in my project, that it do exactly what I wanted and what wasn't working before.

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added the changelog accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by long2ice** [comment]
Thanks for your contribution! So what's the result when you run `make test_oracle` in local?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -14,6 +14,9 @@ Fixed
 ^^^^^
 - Mixin does not work. (#1133)
 - `using_db` wrong position in model shortcut methods. (#1150)
+- Added `schema` attribute to Model's Meta to specify exact schema to use with the model.
```
Comment (long2ice) [inline CHANGELOG.rst:17]: You should format like before, just add `Added` section

**Review by long2ice — APPROVED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (thakryptex) [reply CHANGELOG.rst]: ok, fixed

**Comment by thakryptex** [comment]
> So what's the result when you run `make test_oracle` in local?

For now, I don't have any local Oracle databases, so tests are failing, because of that.

**Comment by long2ice** [comment]
Do you have docker?

**Review by long2ice — APPROVED** [review]

**Comment by thakryptex** [comment]
> Do you have docker?

Yes. Do you have configured Dockerfile for these tests?

**Comment by long2ice** [comment]
Just run this: `docker run -d --network=host -e ORACLE_PASSWORD=123456 --name oracle --restart=always gvenzl/oracle-xe`

**Comment by thakryptex** [comment]
> you run `make test_oracle` in local?

Did it work previously? Maybe I have setup problems with oracle or maybe it didn't work with oracle before, but all of the tests failed.
First errors was something about "You can insert null values into tournament_id" or something.
Then it was "ORA-00955: name is already used by an existing object", so database is not truncated after the tests (or because of test interruption).
So, I can't find the way to test it for now 😅

**Comment by thakryptex** [comment]
@long2ice, wow, you're from Chongqing. I've been there. Great city, I like it 😊

**Comment by long2ice** [comment]
Thanks! So how about this PR? Did oracle run success?

**Comment by long2ice** [comment]
Not worked before, ok, now you just need make the ci pass

**Comment by long2ice** [comment]
Looks like something error, I rerun it

**Comment by thakryptex** [comment]
> now you just need make the ci pass

I did it :)

**Review by long2ice — APPROVED** [review]

**Pull request merged**